snapcraft.yaml

```diff
@@ -932,12 +954,21 @@ parts:
       - lib/*/libnuma*so*
       - lib/*/libpixman*so*
       - lib/*/libusbredir*so*
+      - lib/*/libfdt*.so*
       - share/qemu/keymaps*
-      - share/qemu/efi-virtio.rom*
-      - share/qemu/kvmvapic.bin*
-      - share/qemu/s390-*.img*
-      - share/qemu/slof.bin*
-      - share/qemu/vgabios-*.bin*
+      - share/qemu/efi-virtio.rom
+      - share/qemu/kvmvapic.bin
+      - share/qemu/s390-ccw.img
+      - share/qemu/s390-netboot.img
+      - share/qemu/slof.bin
+      - share/qemu/vgabios-ati.bin
+      - share/qemu/vgabios-bochs-display.bin
+      - share/qemu/vgabios-cirrus.bin
+      - share/qemu/vgabios-qxl.bin
+      - share/qemu/vgabios-ramfb.bin
+      - share/qemu/vgabios-stdvga.bin
+      - share/qemu/vgabios-virtio.bin
+      - share/qemu/vgabios-vmware.bin
```

Comment:
Could we drop the `ati`, `cirrus` and `vmware` ones? For those firmwares to be used, one would need to override the QEMU config and replace the virtio GPU by one of those emulated devices. I can't think of a situation where that would be useful and many of the QEMU CVEs were due to those emulated devices.

The idea would be to limit the user's choice to the saner alternatives GPUs.

Reply:
I mentioned it here but I think it would belong to a follow-up PR if you agree on the principles of course.

Reply:
@simondeziel if we dont need these files we could land a PR now that removes them from the current build and i can then update my PR to reflect that, want to take that one on?

Reply:
https://github.com/canonical/lxd-pkg-snap/pull/284